packages/babel-core/test/fixtures/transformation/misc/regression-2892/expected.js

```diff
@@ -89,7 +89,7 @@ function _foo() {
               return _bar2.apply(this, arguments);
             };
 
-            bar = function bar() {
+            bar = function _ref() {
```

Comment:
This change seems not related to your PR, but i'm not sure why.

Reply:
It is by regenerator: https://github.com/babel/babel/pull/7137